### PR TITLE
feat(i18n-JA): update Japanese language

### PR DESCRIPTION
**Pull request by Suisyan**
Japanese language update.
Signed-off-by: Suisyan 99475190+Suisyan@users.noreply.github.com

### CODE REVIEW

File: src/locales/ja.yaml [hunk 1]
```diff
@@ -38,7 +38,7 @@ app:
       hide_temp_waits: 待機中を隠す
     placeholder:
       command: タブでオートコンプリート、helpと入力でコマンド一覧表示 矢印で履歴ーを表示
-      scroll: 下にスクロール
+      scroll: ''
```
Comment (matmen) [inline src/locales/ja.yaml:41]: What's the reason for this change?

File: src/locales/ja.yaml [hunk 2]
```diff
@@ -53,7 +53,7 @@ app:
     tooltip:
       endpoint_examples: >-
         API URLを入力してください。<br />例としては、<blockquote>fluidd.local,
-        https://192.168.1.150</blockquote><br />があります。
+        https://192.168.1.150</blockquote><br />があります。
```
Comment (matmen) [inline src/locales/ja.yaml:56]: Is this intentional?

File: src/locales/ja.yaml [hunk 3]
```diff
@@ -82,37 +85,51 @@ app:
     title:
       add_dir: ディレクトリの追加
       add_file: ファイルの追加
+      command_palette: コマンドパレット
       download_file: ファイルの取得
+      keyboard_shortcuts: ''
```
Comment (matmen) [inline src/locales/ja.yaml:90]: Is this empty translation intentional?

File: src/locales/ja.yaml [hunk 4]
```diff
@@ -162,32 +185,38 @@ app:
       components_config: 'Moonrakerのプラグイン設定は、こちら <a target="_blank" href="%{url}">です。</a>'
       failed_components: Moonrakerはプラグインに失敗しています。ログを確認し、設定を更新して、moonrakerを再起動してください。
     label:
-      accel_to_decel: 加減速
+      accel_to_decel: Accel to Decel
```
Comment (matmen) [inline src/locales/ja.yaml:188]: What's the reason for this change? Seems like it was translated before

File: src/locales/ja.yaml [hunk 5]
```diff
@@ -162,32 +185,38 @@ app:
...
       acceleration: 加速度
+      accepted_screws: ネジを有効
       actual_time: 実際の
       add_camera: カメラ追加
+      add_category: ''
```
Comment (matmen) [inline src/locales/ja.yaml:193]: Is this empty translation intentional?

File: src/locales/ja.yaml [hunk 6]
```diff
@@ -224,23 +262,33 @@ app:
       total_print_time_avg: プリントあたりの平均値
       total_time: 合計時間
       total_time_avg: プリントあたりの平均値
+      turn_device_off: ' %{device} をオフにする'
+      turn_device_on: ' %{device} をオンにする'
       uncategorized: 未分類
-      unretract_extra_length: アンリトラクト長さ
-      unretract_speed: アンリトラクトスピード
+      unretract_extra_length: Unretract Extra Length
+      unretract_speed: Unretract Speed
```
Comment (matmen) [inline src/locales/ja.yaml:269]: Are these changes intentional? Looks like it was translated before

File: src/locales/ja.yaml [hunk 7]
```diff
@@ -281,36 +335,46 @@ app:
...
       camera: カメラ｜カメラ
       config_files: Configurationファイル
       configure: Configuration
       console: コンソール
+      diagnostics: 診断
+      edit_chart: チャートを編集
       endstops: エンドストップ
       fans_outputs: ファン＆出力
-      gcode_preview: 'Gcode Preview'
+      gcode_preview: 'Gcode Preview [ ベータ版 ]'
```
Comment (matmen) [inline src/locales/ja.yaml:350]: This isn't beta anymore.
```suggestion
      gcode_preview: 'Gcode Preview'
```

File: src/locales/ja.yaml [hunk 8]
```diff
@@ -196,26 +225,35 @@ app:
...
       longest_job: 長い印刷
       low: 低い
       m117: M117
+      moonraker: Moonraker
       name: 名前
       new_password: 新しいパスワード
       no_notifications: 通知なし
       'off': オフ
       'on': オン
+      partial_of_total: '  %{total}の%{partial} '
```
Comment (matmen) [inline src/locales/ja.yaml:238]: I don't think this translation should start with whitespaces
```suggestion
      partial_of_total: '%{total}の%{partial} '
```

File: src/locales/ja.yaml [hunk 9]
```diff
@@ -281,36 +335,46 @@ app:
...
       history: 履歴
       home: ホーム
       jobs: 印刷ジョブ
       limits: プリンター制限
       macros: マクロ
+      metrics_explorer: メトリクス エクスプローラー
       other_files: その他のファイル
-      retract: Firmwareを戻す
+      pending_configuration_changes: 保留中の構成変更
+      retract: Firmware Retraction
```
Comment (matmen) [inline src/locales/ja.yaml:359]: Is there a reason this key is added but not translated?

File: src/locales/ja.yaml [hunk 10]
```diff
@@ -441,25 +554,91 @@ app:
...
       finish: 終了
       update: アップデート
+      update_all: 全てをアップデート
       view_versions: バージョンを表示
     label:
       commit_history: 書込み履歴
       commits_on: 書込みする
       committed: 書込みする
-      dirty: ダーティー
+      dirty: DIRTY
```
Comment (matmen) [inline src/locales/ja.yaml:641]: What's the reason for this change?

**Review by matmen — CHANGES_REQUESTED** [review]

File: src/locales/ja.yaml [hunk 11]
```diff
@@ -52,8 +52,8 @@ app:
       trouble: 'お困りですか？ <a href="%{url}" target="_blank">詳細はこちら</a> をご覧ください。'
     tooltip:
       endpoint_examples: >-
-        API URLを入力してください。<br />例としては、<blockquote>fluidd.local,
-        https://192.168.1.150</blockquote><br />があります。
+        API URLを入力してください。<br />例としては、<br /><blockquote>fluidd.local,
+        https://192.168.1.150 があります</blockquote>。
```
Comment (matmen) [inline src/locales/ja.yaml:56]:
```suggestion
        https://192.168.1.150 があります</blockquote>。
```

File: src/locales/ja.yaml [hunk 12]
```diff
@@ -164,30 +187,36 @@ app:
     label:
       accel_to_decel: 加減速
       acceleration: 加速度
+      accepted_screws: ネジを有効
       actual_time: 実際の
       add_camera: カメラ追加
+      add_category: ''
```
Comment (matmen) [inline src/locales/ja.yaml:193]: This either needs to be removed or translated.

**Review by matmen — CHANGES_REQUESTED** [review]

**Review by matmen — APPROVED** [review]
LGTM, but I'd like to keep this open until all other features for 1.21 are merged

**Comment by matmen** [comment]
@Suisyan Some new translations keys were added with https://github.com/fluidd-core/fluidd/commit/e7106a28fee5c29ac1cc79b0a03f964de344199d, would you be willing to translate those? Thanks!

**Review by pedrolamas — APPROVED** [review]

**Review by matmen — APPROVED** [review]

**Pull request merged**